src/app/pages/news/11.3/02-inscription-view.md

```diff
@@ -1,6 +1,22 @@
 ## Inscription View {#inscriptionView}
 
-Massive design update, browser, finalization ...
+The new Inscription View is finally finished! 
+We've not only successfully migrated all the old SWT-based 
+masks and browsers into the new world but have also given the entire 
+interface an amazing redesign.
+
+- **UI update**:  Prepare to be amazed by the refreshed user interface! 
```

Comment:
I like this claim the most. Totally optional suggestion:

.. .maybe we can enrich it with a sentence, that the UX has been improved with respect to advanced users knowing the tooling since many years, while also making the functionality more appealing for new users.